get_nxpRdLib.sh

```diff
@@ -3,18 +3,6 @@
 blue='\033[0;34m'
 NC='\033[0m' # No Color
 
-prereq() {
-  PREREQS="build-essential cmake $1"
-  
-  if dpkg -s $PREREQS > /dev/null 2>&1; then
-    echo -e "Prerequisites already installed: $PREREQS"
-  else
-    echo -e "[${blue}Installing prerequisites${NC}]"
```

Comment:
It's difficult to find missing dependencies without using the distro-specific package manager.

As an alternative, the script could try running `cmake -h`, `wget -h` and `unzip -h` and in case of an error return a non-zero value.

Also, you could create an [issue template](https://github.com/blog/2111-issue-and-pull-request-templates) that asks users to make sure they have all dependencies installed.